setup.py

```diff
@@ -44,7 +45,7 @@
         "pytest-dependency",
         "pyyaml",
         "requests",
-        "scipy==1.10.1",
+        "scipy==1.14.0",
```

Comment:
If the current works
```suggestion
        "scipy<2",
```